entries/b/buildium.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Buildium": {
+    "domain": "buildium.com",
+    "tfa": [
+      "sms",
+      "totp"
```

```suggestion
      "totp",
      "call"
```
The doc page also mentions 2FA via phone call